globals/boundary_developer_env.go

```diff
@@ -1,8 +1 @@
 package globals
```

Comment:
I guess I could remove this file completely, but I didnt know if we wanted to leave it around in case we create any new developer consts.

Reply:
Happy to leave it for now.